SDK/AppCenter/Microsoft.AppCenter/AppCenter.cs

```diff
@@ -64,5 +64,9 @@ static void PlatformStart(string appSecret, params Type[] services)
         static void PlatformSetCustomProperties(CustomProperties customProperties)
         {
         }
+
+        internal static void PlatformUnsetInstance()
```

Comment:
We need this method on iOS and Windows.Shared as well to fix the build

Reply:
Added.